mendix/README.md

```diff
@@ -0,0 +1,39 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a market [leading low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
+
+With Mendix, you can easily create, change, integrate, test and deploy your applications. Moreover, you can manage branch lines and security as well as extend your app with custom code using the built-in editors.
+
+In addition to the native monitoring capabilities out of the box, Mendix also supports integrations to leading 3rd party application performance management vendors such as DataDog. Refer to our documentation to see all our [supported integrations][3].
```

Comment:
```suggestion
```

Reply:
This is not necessary since customers looking at this information already know an integration exists. Please remove this or reword it to 'Refer to [our documentation][3] for more information about monitoring Mendix with APM tools'.